README.md

```diff
@@ -15,7 +15,7 @@ All this in less than 400 lines of YAML config. View the source in the [lowdefy.
 ## Running this example
 
```

Comment:
This will now be deployed with vercel so we can change all netlify references and instructions to that of vercel please.